web/src/components/console/utils/status-box.tsx

```diff
@@ -75,7 +73,7 @@ export const EmptyBox: React.FC<EmptyBoxProps> = ({ label }) => {
   return (
     <Box>
       <div data-test="empty-message" className="pf-u-text-align-center">
-        {label ? t('public~No {{label}} found', { label }) : t('public~Not found')}
+        {label ? t('~No {{label}} found', { label }) : t('Not found')}
```

Comment:
```suggestion
        {label ? t('No {{label}} found', { label }) : t('Not found')}
```

Reply:
Applied the suggested changes in the latest commit.